src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
+      <nav>
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              href="#apple"
+              class="header__link is-active"
```

Comment:
Fix class everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#apple"
              class="nav__link is-active"
```

Reply:
Classes were changed everywhere.